### DISCUSSION TITLE
New to Flopy/ModFlow trying to learn how CHD calculates head values

### DISCUSSION THREAD

**Discussion by AH-CSM** [modflowpy/flopy#1413] [Q&A] [answered]
I am working with Flopy to model groundwater table and head changes over time. My current code employs a CHD package. I am being returned head result arrays that have a stated starting head and ending head. However I am wanting a model that doesn't necessarily have an ending head of specifically zero. I am trying to build a model that moves towards a lower head but I want to see how long it takes to get there. Regardless of how I specify the time steps and lengths, my heads always end at zero. I clearly do not understand how CHD works or if this is the best way to model what I am looking at. Any advice or ideas on how CHD calculates heads over time and arrays or how to approach this problem would be greatly appreciated.

**Comment by jlarsen-usgs** [comment, marked as answer]
@AH-CSM 

I'd personally approach this problem using the GHB (general head boundary) condition package instead of CHD. With GHB you could set the head on the boundary close to the bottom elevation of your model and then set the conductance accordingly. Using this approach it should be relatively easy to set up and simulate drainage from your model.

**Reply by AH-CSM** [reply]
Thank you for the suggestion. I did try this approach and I have a working code for it that is similar to the "MODFLOW Tutorial 2: Unconfined Transient Flow Model" on the flopy website based on Modflow6 instead of 2005. However, the model is approaching steady-state almost instantly it appears and no reduction of period length or hydraulic conductivity is showing any kind of progression towards the equilibrium. Assuming the model follows hydraulic predictions, I should see some kind of slow progression towards the lower head over the x dimension and reach steady-state at different specific times. I am wanting to measure the time it takes the model to get there, but if it is happening instantly, then my model is not working correctly. Any suggestions?

**Reply by cnicol-gwlogic** [reply]
There's usually a "switch" point conductance below which you will see a delay effect. Above that, you'll see no change. Try to reduce the conductance more. I'd start the conductance value as calculated using a conceptually reasonable low K, along with cell geometry and a unit length. Then proceed to cut the K value back from there (by an order of magnitude for example). 

But having said that, I'd consider more deeply what you are trying to model. It sounds to me like a boundary condition might not be the best solution. An applied flux (recharge) might be conceptually better. Eg: if recharge drops from something to nothing, how long do heads take to stabilise at a new equilibrium? But I don't really understand what question you are trying to answer, so it is difficult to provide suggestions.

**Reply by AH-CSM** [reply]
My research question is simple: I have a geological area that has had a rise in the water table and I want to model how long it would take to reduce that water table given a change in hydraulic conductivities. If I know it is starting at a higher value of say 10m, and want to reduce it to 2m instead, what hydraulic conductivity would best decrease it over time and how long might it take for it to reach the 2m

**Reply by RyanConway91** [reply]
Sounds like you trying to simulate the transient drainage of groundwater mound. In the simplest form, you don't need any boundary conditions to "force" the head dissipation. This just happens due to gravity. 

A dummy model of this would like something like:
- Set CHDs/GHBs far from the mound to a steady state regional value. This is the head that the model will end up at.  
- Set your initial "elevated" head distribution (or find a BC that produces the head distribution you want)
- Run the model as transient with many timesteps. The mound will drain through time. You can change the K values and see how this effects the drainage time.
